src/pcf857x.ts

```diff
@@ -230,9 +239,14 @@ export abstract class PCF857x<PinNumber extends PCF8574.PinNumber | PCF8575.PinN
       // decrease the use count of the GPIO and unexport it if not used anymore
       this._gpio['pcf857xUseCount']--;
       if (this._gpio['pcf857xUseCount'] === 0) {
+        // error-proof in case somehow a negative value for gpioPin was somehow allowed during enableInterrupt() call
+        if (this._gpioPin >= 0) {
```

Comment:
According to my comment above:
```ts
if (this._gpioPin !== null) {
```

Also negative gpioPin numbers may be possible here in the `PCF857x._allInstancesUsedGpios` object 
(but will of course error in the onoff module).

Reply:
It's not an array, it's an object:
https://github.com/crycode-de/node-pcf8574/blob/534506ec9e02423dae7b79f5806ab37c5c803c5d/src/pcf857x.ts#L96-L97
The keys in objects can be a number or a sting.